src/property-filter/filtering-token/styles.scss

```diff
@@ -14,6 +14,8 @@ $token-grouped-padding-block: 2px;
 $token-grouped-padding-inline: styles.$control-padding-horizontal;
 $inner-token-padding-block: 1px;
 $inner-token-padding-inline: styles.$control-padding-horizontal;
+$border-radius-token: awsui.$border-radius-token;
```

Comment:
Why do we redeclare it here?

Reply:
Just for consistency with border-radius-inner-token